doc/source/reference/apis/metadata.md

```diff
@@ -22,7 +26,97 @@ this feature graduates. If you have any comments or suggestion please open the i
 
 ## Model Metadata (incubating)
 
-With Seldon you can easily define metadata for your models that is compatible with [kfserving dataplane proposal](https://github.com/kubeflow/kfserving/blob/master/docs/predict-api/v2/required_api.md#model-metadata) specification:
+With Seldon you can easily add metadata to your models.
+You can add metadata using one of two formats:
+- `v1` format that closely correlate to the current structure of `SeldonMessage`
+- `v2` format that is future-proof and fully compatible with `kfserving` dataplane proposal
+
+As you will see in following sections difference is minimal and mostly relates to the input/output format.
+
+
+### V1 DataPlane
+
+In order to use `v1` metadata format you need to specify `apiVersion: v1`.
+
+#### Array input/output
+```YAML
+apiVersion: v1
+name: my-model-name
+versions: [ my-model-version-01 ]
+platform: seldon-custom
+inputs:
```

Comment:
nit: since the `v1` version of the metadata only accepts a single input and a single output, should we just call these fields `input` and `output`?

Reply:
I don't mind either, left plural to only diff from v2 on the values, not keys.
@axsaucedo What do you think?